src/DssSpell.sol

```diff
@@ -165,6 +165,7 @@ contract DssSpellAction is DssAction {
             DssExecLib.setContract(D3M_COMPOUND_POOL, "king", address(this));
 
             DssExecLib.authorize(D3M_COMPOUND_PLAN, D3M_MOM);
+            DssExecLib.authorize(D3M_COMPOUND_PLAN, DssExecLib.esm());
```

Comment:
I was actually checking yesterday if was necessary to rely ESM in any of the D3M modules and didn't find any reason. Why to the plan? Plan won't be used at all if `vat.live == 0`.

Reply:
I'm not very familiar with the D3M anymore. It was recommended above to consider adding. I'll leave it in or out based on how you guys modeled this.

Reply:
Some integration can rely on the plan target value for something and malicious governance can maybe manipulate it (the plan is the only file that doesn't block auth operations on `!vat.live`).
Also the plan can change in a way that it will be more harmful in the future and if the ESM is not relied then because the code was reused it can be problematic.
Isn't it the regular practice that when there is authed operations to other block on `!vat.live` or rely the ESM?

Reply:
What do you mean with some integration? I think we should only rely the esm where malicious governance can actually provoke harm after ES.

Reply:
What I had in mind is that malicious governance can file a new `tack` and skew the result of `getTargetAssets`.
For example it can skew it to saying that Maker should deposit a lot of funds. Then some external dapp that for some reason also relies on `getTargetAssets` to imitate/front-run/predict Maker actions may pre-borrow a lot of funds with a bad borrow rate.
It is pretty weird scenario though, so if you think we shouldn't worry too much about these kind of things I'm fine with that.

Reply:
(do note that there is a comment above the plan's `getTargetAssets` directed to on chain integrations that just call that view function)

Reply:
Yeah honestly not sure if we should care about that scenario, relying the ESM has a pretty specific objective which is protecting MakerDAO system from malicious governance actions after ES.
`D3MHub` also has a `file` function that is not under `vat.live` but it is harmless for the system behaviour as the whole `plan` is.
But yeah I see your point which being useful for a third party integration, however I'm not convinced we should be adding this and all the future plans to the esm keeper tasks just due to that.

Reply:
@godsflaw @brianmcmichael @kmbarry1 would like to see your opinion here

Reply:
Hmmmm. I'm not particularly swayed by the argument of trying to prevent hypothetical harm to external systems. Those generally should be implementing their own mitigations for Maker going off the rails. One thing we can't rule out is that in the future, though, some other part of the _Maker system itself_ will be relying on `getTargetAssets`. But, this authorization could always be added in the future if that became the case. So, I personally wouldn't put it in, as we should try to limit these authorizations to things that absolutely need to have governance `denied` to protect the protocol in a malicious scenario if only for organizational purposes.

Reply:
Going ahead and removing it unless there is a strong opinion in the other direction.

Reply:
done

Reply:
For the record I agree.  The point of the ESM `deny()` is to rug malicious governance from harming the Maker Protocol.  Given the ESM triggers a `vat.live = 0` we already know the plan isn't even considered in the `exec()` as that short circuits to `unwind()`.  So, the only other reason is altruism to prevent someone else that is reading those calculations.  I did check `dss-cron` for the job that runs this, and there is no dependency on the `ilk.plan` variables either.